Apps/CommonData/src/ErrorHandling/ExceptionUtility.cs

```diff
@@ -0,0 +1,46 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Common.Data.ErrorHandling
+{
+    using System.Net;
+    using System.Runtime.CompilerServices;
+
+    /// <summary>
+    /// Utility for throwing exception.
+    /// </summary>
+    public static class ExceptionUtility
```

Comment:
Should this be a factory?

Reply:
@sslaws wrote this for WaitingQueueWeb.  Are you asking it to be renamed?